src/xml2csv.py

```diff
@@ -1,90 +1,81 @@
-import csv
-import gc
 import json
-import os
 import sys
+from csv import DictWriter
 from datetime import datetime
-from tempfile import TemporaryDirectory
+from io import StringIO
 
-import numpy as np
 from bs4 import BeautifulSoup
```

Comment:
My bad, you're still using it. https://github.com/datasciencebr/serenata-de-amor/pull/23/files#diff-986bb1fc5b0bcbb749606f7613c049e9R43